tests/materials_test.cpp

```diff
@@ -45,9 +45,9 @@ TEST_CASE( "Resistance_vs_material_portions", "[material]" )
     REQUIRE( mostly_plastic.get_base_material().id == material_plastic );
 
     CHECK( mostly_steel.resist( damage_cut ) > mostly_plastic.resist( damage_cut ) );
-    CHECK( mostly_steel.resist( damage_acid ) == mostly_plastic.resist( damage_acid ) );
+    CHECK( mostly_steel.resist( damage_acid ) > mostly_plastic.resist( damage_acid ) );
```

Comment:
This change (and all others to material test) appear unrelated, and should be separated.

Reply:
Oops, those got in from another PR. Thanks for catching that.